src/heranca_exemplos/marcas_de_veiculos/Base.java

```diff
@@ -1,9 +1,7 @@
 package heranca_exemplos.marcas_de_veiculos;
 
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Scanner;
 
 public class Base {
```

Comment:
Sobre o atributo abaixo
public static Scanner scanner = new Scanner(System.in);

Se for usado apenas na classe Base pode ser privado
Se for usado nas heranças da classe base pode ser protected
Só deve ser public se realmente qualquer outra classe em qualquer outro pacote do projeto vai invocar